### ISSUE TITLE
Combining two graph types

### ISSUE THREAD

**Issue by RamonBeast** [reactjs/react-chartjs#79]
Since chart-js 2 is not yet supported, with the current version of react-chartjs is there a way to combine two charts into the same plot? e.g.: a bar and a line.

**Comment by anilasj** [comment]
Yes I'm interested in this as well. We're looking to combine the line and bar chart together.

**Comment by austinpray** [comment]
I would make suggestions on https://github.com/jhudson8/react-chartjs/pull/87 to see if this feature can be added.

**Comment by kschrute** [comment]
Any updates on this?

**Comment by diegocam** [comment]
Any updates on this?
